includes/CMB2_hookup.php

```diff
@@ -294,6 +329,50 @@ public function return_column_display( $empty, $custom_column, $object_id ) {
 	}
 
 	/**
+	 * [add_context_metabox description]
+	 */
+	public function add_context_metabox() {
+
+		if ( ! $this->show_on() ) {
+			return;
+		}
+
+		$form = cmb2_get_metabox( $this->cmb->prop( 'id' ) );
+
+		$form->show_form();
+	}
+
+	/**
+	 * Add an additional CSS class for the form box in the "after title" hook.
+	 *
+	 * @param array $classes Array of classes for the cmb2-wrap.
+	 * @param CMB2  $cmb     This CMB2 object.
+	 */
+	public function add_context_class( $classes, $cmb ) {
+
+		if ( ! $this->show_on() ) {
+			return;
+		}
+
+		// If we aren't applying one of our extended contextes, just return our array.
+		if ( empty( $cmb->prop( 'context' ) ) || ! in_array( $cmb->prop( 'context' ), array( 'form_top', 'before_permalink', 'after_title', 'after_editor' ) ) ) {
+			return $classes;
+		}
+
+		// Include a generic context wrapper and the postbox wrapper.
+		$classes[] = 'postbox cmb2-context-wrap';
```

Comment:
the `postbox` class ensures it looks like the other metaboxes. I can create a conditional if you'd prefer.